src/About.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+	Content,
+	Container,
+	Section,
+	Heading,
+	Tile,
+	Notification,
+} from 'react-bulma-components';
+
+const About = () => {
+  return (
+		<Container>
+			<Section>
+				<Tile kind="ancestor" id="about">
+					<Tile kind="parent">
+						<Tile
+							kind="child"
+							renderAs={Notification}
+							color={"gray"}
+						>
+							<Heading weight="semibold" subtitle style={{ textAlign: 'center' }}>About Service</Heading>
+							<Content>
+								<p>Archive.NeoFS is a web application that allows users to create blockchain archives of any span (from block 0 to the current block or a custom range) directly in the browser. It operates fully client-side, leveraging streaming techniques to efficiently fetch and store blockchain data without requiring additional backend processing.</p>
+								<p>The service supports four networks: mainnet, testnet, NeoFS mainnet, and NeoFS testnet. It interacts with the <a href="https://github.com/nspcc-dev/neofs-rest-gw/" target="_blank" rel="noopener noreferrer">NeoFS REST gateway</a> to retrieve blockchain data stored in NeoFS objects and assembles them into a structured archive format (.acc), optimized for chain data storage.</p>
+								<p><a href="https://github.com/nspcc-dev/send-fs-neo-org" target="_blank" rel="noopener noreferrer">Frontend part</a> first determines the latest available block in the selected network. Blockchain data is stored in NeoFS objects, indexed in segments of 128,000 blocks. The service sequentially retrieves these index files, extracts block references, and downloads the corresponding data. The entire process runs in the browser, streaming blocks directly into an archive file, ensuring efficient storage and real-time access.</p>
```

Comment:
> determines the latest available block

How?

> indexed in segments of 128,000 blocks

Indexes and objects are separate things, this should be described. In fact it's one object per one block, but index objects are needed to map indexes to OIDs.